.gitpod.yml

```diff
@@ -0,0 +1,29 @@
+image: gitpod/workspace-full:latest
+
+tasks:
+  - name: Setup
+    init: |
+      make deps
+    command: |
+      gp sync-done setup
+      exit 0
+  - name: Run frontend
+    command: |
+      gp sync-await setup
+      make watch-frontend
+  - name: Run backend
+    command: |
+      gp sync-await setup
+      export GITEA__server__ROOT_URL="https://3000-$GITPOD_WORKSPACE_ID.${GITPOD_WORKSPACE_CLUSTER_HOST}/"
+      export TAGS="sqlite sqlite_unlock_notify"
+      make watch-backend
+
+vscode:
+  extensions:
+    - editorconfig.editorconfig
+    - dbaeumer.vscode-eslint
+    - golang.go
```

Comment:
Could you also add https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint?

Reply:
Also, https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

Reply:
done.